packages/labextension/package.json

```diff
@@ -29,10 +29,12 @@
   "typings": "lib/index.d.ts",
   "scripts": {
     "build": "npm run build:lib && npm run build:labextension",
-    "build:dev": "npm run build:lib",
-    "build:labextension": "rimraf dist && mkdirp dist && cd dist && npm pack ..",
+    "build:dev": "npm run build:lib && jupyter labextension build --development True .",
+    "build:labextension": "rimraf ../../nbdime/labextension && mkdirp ../../nbdime/labextension/federated && jupyter labextension build . && cd ../../nbdime/labextension && npm pack ../../packages/labextension",
```

Comment:
The non-federated version of the extension shouldn't need to be disted in the python package (it cannot be picked up in any way), so we can save quite a few MB by not putting it in.